sdk/trace/src/jmh/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorBenchmark.java

```diff
@@ -33,13 +33,13 @@
 @State(Scope.Benchmark)
 public class BatchSpanProcessorBenchmark {
 
-  private static class DelayingSpanExporter implements SpanExporter {
+  public static class DelayingSpanExporter implements SpanExporter {
```

Comment:
if we want to re-use this, please pull it up out of this class to the top level, so it's clear it's a reusable component for benchmarking.